test/interactive_tarantool.lua

```diff
@@ -184,6 +183,14 @@ function mt.execute_command(self, command, opts)
     self:_assert_command_echo(prepared_command, {deadline = deadline})
 end
 
+function mt.execute_continued_command(self, command, opts)
+    local base_prompt = self._prompt
+    local continued_prompt = string.rep(' ', #base_prompt - #'> ') .. '> '
+    mt.set_prompt(self, continued_prompt)
+    mt.execute_command(self, command, opts)
+    mt.set_prompt(self, base_prompt)
+end
```

Comment:
It is not used in this patchset. Let's propose it for review in a corresponding patchset.

Reply:
You are right, removed.